<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiEditBoundaryConditions</class>
 <widget class="QDialog" name="GuiEditBoundaryConditions">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>edit boundary conditions</string>
  </property>
  <property name="windowIcon">
   <iconset resource="engrid.qrc">
    <normaloff>:/icons/resources/icons/G.png</normaloff>:/icons/resources/icons/G.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_BoundaryConditions">
      <attribute name="title">
       <string>Boundary conditions</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="T">
         <column>
          <property name="text">
           <string>BC-index</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>BC-name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>BC-type</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="8">
          <widget class="QPushButton" name="pushButtonAllOff">
           <property name="text">
            <string>all off</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QPushButton" name="pushButtonAllB">
           <property name="text">
            <string>all &quot;B&quot;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="pushButtonDelete">
           <property name="text">
            <string>delete</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QPushButton" name="pushButtonAllA">
           <property name="text">
            <string>all &quot;A&quot;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>new volume</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditVolume"/>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButtonAdd">
           <property name="text">
            <string>add</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_BoundaryTypes">
      <attribute name="title">
       <string>Boundary types</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="1" column="2" rowspan="2">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>parameters</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTableWidget" name="tableWidgetPBC">
              <column>
               <property name="text">
                <string>name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>value</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>manage boundary conditions</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="1" column="2">
             <widget class="QPushButton" name="pushButtonAddBoundaryType">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBoxBoundaryType">
              <item>
               <property name="text">
                <string>wall</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>inviscid wall</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>inlet</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>outlet</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>symmetry</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="0" colspan="3">
             <widget class="QListWidget" name="listWidgetBoundaryType">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QPushButton" name="pushButtonDeleteBoundaryType">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="lineEditBoundaryType"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Solver">
      <attribute name="title">
       <string>Solver</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0" colspan="2">
        <layout class="QVBoxLayout" name="verticalLayout_Solver"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>MPI configuration</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" rowspan="5">
        <widget class="QTableWidget" name="tableWidget_Processes">
         <column>
          <property name="text">
           <string>Host</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Weight</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_AddProcess">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_RemoveProcess">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_ImportHostFile">
         <property name="text">
          <string>Import hostfile...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_ExportHostFile">
         <property name="text">
          <string>Export hostfile...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>534</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>T</tabstop>
  <tabstop>lineEditVolume</tabstop>
  <tabstop>pushButtonAdd</tabstop>
  <tabstop>pushButtonDelete</tabstop>
  <tabstop>pushButtonAllA</tabstop>
  <tabstop>pushButtonAllB</tabstop>
  <tabstop>pushButtonAllOff</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>lineEditBoundaryType</tabstop>
  <tabstop>comboBoxBoundaryType</tabstop>
  <tabstop>pushButtonAddBoundaryType</tabstop>
  <tabstop>listWidgetBoundaryType</tabstop>
  <tabstop>pushButtonDeleteBoundaryType</tabstop>
  <tabstop>tableWidgetPBC</tabstop>
  <tabstop>tableWidget_Processes</tabstop>
  <tabstop>pushButton_AddProcess</tabstop>
  <tabstop>pushButton_RemoveProcess</tabstop>
  <tabstop>pushButton_ImportHostFile</tabstop>
  <tabstop>pushButton_ExportHostFile</tabstop>
 </tabstops>
 <resources>
  <include location="engrid.qrc"/>
  <include location="engrid.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>GuiEditBoundaryConditions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>GuiEditBoundaryConditions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
